packages/custom-range/src/custom-range.ts

```diff
@@ -1,66 +1,50 @@
 class CustomRangeIndicator {
-    private rangeContainer: HTMLElement
-    private rangeElement: HTMLInputElement
-    private indicatorElement: HTMLSpanElement
-    private indicatorClassname: string[]
+    private rangeContainer: HTMLElement;
+    private rangeElement: HTMLInputElement;
+    private indicatorElement: HTMLSpanElement;
+    private indicatorClassname: string[];
 
     constructor(containerElement_: string | HTMLElement, rangeIndicator?: string) {
-        const containerElement = typeof containerElement_ === "string" ? document.querySelector(`${containerElement_}`) : containerElement_
-        if (!(containerElement instanceof HTMLElement)) throw new Error("No valide container element")
+        const containerElement = typeof containerElement_ === "string" ? document.querySelector(containerElement_) : containerElement_;
+        if (!(containerElement instanceof HTMLElement)) throw new Error("No valid container element");
 
-        this.rangeContainer = containerElement
-        this.rangeElement = this.rangeContainer.querySelector("[data-input-range]") as HTMLInputElement
+        this.rangeContainer = containerElement;
+        this.rangeElement = this.rangeContainer.querySelector("[data-input-range]") as HTMLInputElement;
 
-        if (!(this.rangeElement instanceof HTMLInputElement)) throw new Error("The provided element doesn't have a valid HTMLInputElement. Make sure to add data-input-range to the input element")
+        if (!(this.rangeElement instanceof HTMLInputElement)) throw new Error("The provided element doesn't have a valid HTMLInputElement. Make sure to add data-input-range to the input element");
 
-        this.indicatorClassname = rangeIndicator?.split(" ") || (this.rangeContainer.dataset.rangeIndicator)?.split(" ") || []
-        this.indicatorElement = this.initIndicator()
-        this.initRange()
-        this.updateIndicatorSize()
+        this.indicatorClassname = rangeIndicator?.split(" ") || (this.rangeContainer.dataset.rangeIndicator)?.split(" ") || [];
+        this.indicatorElement = this.initIndicator();
+        this.initRange();
+        this.updateIndicatorSize();
     }
 
     private initIndicator() {
```

Comment:
To make things easier, it's always best not to apply predefined CSS classes at this level. [Here's an example](https://github.com/unify-ui-dev/flexilla/blob/main/examples/vanilla/custom-range/index.html), using UnoCSS (Chrome only currently). As shown in the example, we provide classes to apply to the custom indicator either through options or data attributes. If possible, please revert to the previous logic for this.

The CSS style you provided is good and will be added as a ready-to-use file for styling the input range. Keep the style as it is, and we'll look at providing usage examples later on.